packages/base/src/mwc-base-element.ts

```diff
@@ -0,0 +1,84 @@
+/**
+@license
+Copyright 2018 Google Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import {LitElement} from '@polymer/lit-element/lit-element.js';
+import {TemplateResult} from 'lit-html';
+export {TemplateResult};
+export {html, property, customElement, query} from '@polymer/lit-element/lit-element.js';
+export {classMap} from 'lit-html/directives/classMap.js';
+export {observer} from './observer.js';
+
+export abstract class MWCBaseElement extends LitElement {
```

Comment:
How about just `BaseElement`. I don't think we need `MWC` on any of the class names.